src/Config.cc

```diff
@@ -575,15 +583,15 @@ std::vector<ZyppFlags::CommandGroup> Config::cliOptions()
               _("Additionally use disabled repositories providing a specific keyword. Try '--plus-content debug' to enable repos indicating to provide debug packages.")
         },
         { "disable-repositories", 0, ZyppFlags::NoArgument, std::move(ZyppFlags::BoolType( &disable_system_sources, ZyppFlags::StoreTrue )
-              .after([](){
-                MIL << "Repositories disabled, using target only." << endl;
-                Zypper::instance().out().info(
-                  _("Repositories disabled, using the database of installed packages only."),
-                  Out::HIGH);
-              })),
+              .after([ notifyDisableRepos ](){ notifyDisableRepos(); })),
               // translators: --disable-repositories
               _("Do not read meta-data from repositories.")
         },
+        // synonym for --disable-repositories for SUMA
```

Comment:
Please add a thet this can be reverted after SUMA updated their SALT stack.

Reply:
Done